orchestration/dagster_orchestration/hca_orchestration/tests/e2e/conftest.py

```diff
@@ -99,7 +99,7 @@ def load_hca_run_config(dataset_name: str, dataset_info: DatasetInfo):
             },
             "scratch_config": {
                 "config": {
-                    "scratch_bucket_name": "broad-dsp-monster-hca-dev-test-storage",
+                    "scratch_bucket_name": "broad-dsp-monster-hca-dev-temp-storage",
```

Comment:
Our test data should always go in the temp storage bucket so it'll get auto deleted after 7 days. 🤦